src/redux/books/books.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+
+const Books = (props) => {
+  const { genre, title, author } = props;
+
+  return (
```

Comment:
- [ ] OPTIONAL - Kindly place the Book component in the components folder. The redux folder should only contain reducers and the redux store.